app/core/api/cms-api.service.ts

```diff
@@ -22,6 +24,13 @@ export class CmsApiService {
   public getUrl(apiUrl: string, headers: any) {
     return this.http.get(this.cmsUrl + apiUrl, headers);
   }
+  
+  public _getUrl(apiUrl: string) {
+    let headers = new Headers();
+    headers.append('authorization', 'Bearer ' + localStorage.getItem('id_token'));
+    headers.append('Access-Control-Allow-Origin', '*');
+    return this._http.get(this.cmsUrl + apiUrl, {headers});
+  }
```

Comment:
can you rename the function so it is clear what the function does? for example `getUrlWithForAttachments`